src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -1,12 +1,19 @@
 package racingcar.domain
 
 data class Car(
+    val name: String,
     val position: Int = 1,
 ) {
 
+    init {
+        if (name.length > 5) {
+            throw IllegalArgumentException("자동차 이름은 5자 이하만 가능합니다.")
+        }
+    }
+
     fun move(point: Int): Car {
         if (point >= 4) {
-            return Car(position + 1)
+            return Car(this.name, position + 1)
```

Comment:
Car가 data class이니 copy 함수를 활용해볼 수 있을 것 같아요. 😃

```suggestion
            return copy(position = position + 1)
```

Reply:
오우 이런기능이!!